docs/nx/next/init-deployment.md

```diff
@@ -87,38 +90,40 @@ As a rule of thumb, each task here references a target execution via Nx defined
 
 ```yaml
 helm:
-    description: Lint Helm Charts
-    command:
-      - npx nx affected --base="$BASE_SHA" --target=helm-lint
+  description: Lint Helm Charts
+  command:
+    - npx nx affected --base="$BASE_SHA" --target=lint
 ```
 
 `apps/myapp/project.json`
 
 ```yaml
-"helm-lint":
+"lint":
   {
     "executor": "nx:run-commands",
     "options":
       {
-        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
+        "commands": [{ "command": "lint", "forwardAllArgs": false }],
         "cwd": "apps/myapp/build/helm",
```

Comment:
I missed this one initially, but the cwd has to point to the helm chart, in the code currently `${helmProjectPath}/build/helm`

Reply:
true, updated

Reply:
@MetecanAydin - Have you pushed this update?

Reply:
@Danphillipz yes, should be there